visualizer/options-menu.js

```diff
@@ -61,6 +79,37 @@ class OptionsMenu extends HtmlContent {
       }
     })
 
+    // preferences
+    this.d3Preferences = this.d3OptionsList.append('div')
+      .classed('section preferences', true)
+    this.d3Preferences.append('h2')
+      .text('Preferences')
+
+    const prefUl = this.d3Preferences.append('ul')
+    const prefLi = prefUl.selectAll('li').data(preferences)
+    prefLi.exit().remove()
+    prefLi.enter().append('li')
+      .html(d => {
+        return (`
+          <div class='overflow-wrapper'>
+            <label>
+              <input id='${d.id}' type='checkbox' "${d.value ? 'checked' : ''}" /> 
+              <span class='icon-wrapper'>
+                <img class="icon-img checked" data-inline-svg src="/visualizer/assets/icons/checkbox-checked.svg" />
+                <img class="icon-img unchecked" data-inline-svg src="/visualizer/assets/icons/checkbox-unchecked.svg" />
+              </span>
+              <span class='copy-wrapper'>${d.title}</span>
+            </label>
+          </div>
+        `)
+      })
```

Comment:
I think this is basically identical to the tree generated by `addFgOptionCheckbox`, only appended to a different parent element. Maybe `addFgOptionCheckbox` could be reused here with some slight tweaks? eg. get pass in the parent as an option or something